src/arrayMethodSort.js

```diff
@@ -4,9 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compareStrings = (prev, current) => {
+    if (String(prev) > String(current)) {
+      return 1;
+    }
+
+    if (String(prev) < String(current)) {
+      return -1;
+    }
+
+    return 0;
+  };
```

Comment:
For your solution will be enough to return the boolean value `String(prev) > String(current)`. True == 1, False == 0, so your if condition inside `sort2` will work ok.